app/helpers/support_time_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module SupportTimeHelper 
```

Comment:
I think these helpers should be nested in a support directory to follow convention of other parts. `app/helpers/support/time_helper.rb` then you'd have:

```
module Support
  module TimeHelper
  end
end
```